Tools/jit/_llvm.py

```diff
@@ -32,12 +32,19 @@ async def wrapper(
     return wrapper
 
 
-_CORES = asyncio.BoundedSemaphore(os.cpu_count() or 1)
+_CORES: asyncio.BoundedSemaphore | None = None
+
+
+def _get_cores() -> asyncio.BoundedSemaphore:
```

Comment:
I'm no asyncio expert so maybe there's a better way to do this... but after poking around at this, I believe the issue here is that we end up calling `.build()` twice for FAT builds, once per triple, which each spin up their own event loop. However, because this was globally instantiated, `_CORES` gets tied to the first event loop, so trying to use it in the second event loop results in an error.

Now, we can lazily instantiate `_CORES` when it's needed in the current event loop.